volumes/vds/sql/views/create-view-detector_inventory.sql

```diff
@@ -0,0 +1,91 @@
+DROP VIEW vds.detector_inventory;
```

Comment:
I feel like some of these rules should be in look up tables instead, to make it a bit easier to be understood. 

Expected bins and RESCU detector codes translating to different areas for starters.